pkg/manager/task_export.go

```diff
@@ -106,7 +109,8 @@ func (t *ExportTask) ExportScenes(ctx context.Context) {
 		}
 
 		newSceneJSON.Performers = t.getPerformerNames(performers)
-		newSceneJSON.Tags = t.getTagNames(tags)
+	
+	    newSceneJSON.Tags = t.getTagNames(tags)
```

Comment:
Fix indent

Reply:
Fixed

Reply:
You removed a line here but did not fix the indent.

Reply:
This is not fixed.

Reply:
Fixed from github.